.eslintrc

```diff
@@ -6,7 +6,9 @@
       {
         "capIsNew": false
       }
-    ]
+    ],
+    "jsx-a11y/no-static-element-interactions": 0,
+    "no-console": 0
```

Comment:
Let's leave `no-console` as a warning, and move the `no-static-element-interactions` rule to the `.eslintrc` file in the client directory